lua/autorun/server/sv_spawn_protection.lua

```diff
@@ -6,6 +6,9 @@ local spawnProtectionMoveDelay = 1
 -- Time in seconds before spawn protection wears off if no action is taken
 local spawnProtectionDecayTime = 10
 
+-- How long players are allowed to hold weapons after spawning (in seconds)
+local spawnProtectionWeaponGracePeriod = 0.001
```

Comment:
This opens up liability, it does. I can't possibly do it by hand, but I don't know how to replicate how someone would hack it.

My guess is the timing is so low that this will only allow for 1 tick/frame

Reply:
![image](https://user-images.githubusercontent.com/22619972/43437816-9893730a-9450-11e8-9bd4-70e2724d90ac.png)